docs/rules/define-macros-order.md

```diff
@@ -27,85 +27,85 @@ This rule reports the `defineProps` and `defineEmits` compiler macros when they
 
 - `order` (`string[]`) ... The order of defineEmits and defineProps macros. You can also add `"defineOptions"` and `"defineSlots"`.
 
-### `{ "order": ["defineProps", "defineEmits"] }` (default)
+### `{ "order": ["defineOptions", "defineProps", "defineEmits", "defineSlots"] }` (default)
```

Comment:
Why is this order the default? Can you link to examples in the official Vue docs that use this order? Or some explanation?

Reply:
@FloEdelmann Currently, the `eslint-plugin-vue` doc has two defaults, which seems to be done by mistake:
![image](https://github.com/vuejs/eslint-plugin-vue/assets/28700378/b4e37fe1-0b8b-45e9-ae2b-314c30462508)

Reply:
It is indeed a mistake in the docs that the second example is marked as "default". That should be fixed in any case, ideally in another PR to fix it without changing the default. Would you fancy doing that?

However, changing the default should only be done if it fits most use cases and/or is the official recommendation from the Vue docs. So can you link to any of those?

Reply:
Ah, I see. I thought the second one is meant to be the new default, so I adjusted the default order to reflect that, I've pushed a new commit to remove the false default order, please take a look.